api/requirements-dev.txt

```
@@ -45,7 +45,7 @@ packaging==23.2
     #   pyproject-api
     #   pytest
     #   tox
-platformdirs==3.11.0
+platformdirs==4.0.0
```

Downgrade platformdirs to meet other dependencies requirements.
```suggestion
platformdirs==3.11.0
```